recipe/meta.yaml

```
@@ -1,40 +1,75 @@
 {% set name = "amqp" %}
-{% set version = "5.0.6" %}
+{% set version = "5.3.1" %}
+{% set tests_to_skip = " test_read_frame__windowstimeout " %}
+{% set tests_to_skip = tests_to_skip + " or test_heartbeat_check_rate_default" %} # [win]
+{% set tests_to_skip = tests_to_skip + " or test_heartbeat_check_rate_four" %} # [win]
+{% set tests_to_skip = tests_to_skip + " or test_heartbeat_check_rate_wrong" %} # [win]
 
 package:
     name: {{ name|lower }}
     version: {{ version }}
 
 source:
-    fn: {{ name }}-{{ version }}.tar.gz
-    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: 03e16e94f2b34c31f8bf1206d8ddd3ccaa4c315f7f6a1879b7b1210d229568c2
+    - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+      sha256: cddc00c725449522023bad949f70fff7b48f0b1ade74d170a6f10ab044739432
+    - url: https://github.com/celery/py-{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+      sha256: 865f165c1ff27052e5a6af249f42659b8fa21d25279a6a8a152580fca3b0de36
+      folder: gh_src
 
 build:
-    noarch: python
+    script_env:
+        - CELERY_ENABLE_SPEEDUPS
```

```suggestion
        - CELERY_ENABLE_SPEEDUPS=true
```
see https://github.com/celery/py-amqp/blob/main/README.rst#speedups and https://github.com/celery/py-amqp/blob/v5.3.1/.github/workflows/ci.yaml#L46C11-L46C35.

But the docs say it's experimental even after 5 years https://github.com/celery/py-amqp/pull/311. The speedup is only around 50% in usecs. That means we can discard it now, but leave a comment for our future selves to decide if we can enable it again.